pkgs/servers/sql/postgresql/ext/postgis.nix

```diff
@@ -34,9 +35,9 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     sed -i 's@/usr/bin/file@${file}/bin/file@' configure
-    configureFlags="--datadir=$out/share/postgresql --datarootdir=$out/share/postgresql --bindir=$out/bin --with-gdalconfig=${gdal}/bin/gdal-config --with-jsondir=${json_c.dev}"
+    configureFlags="--datadir=$out/share/postgresql --datarootdir=$out/share/postgresql --bindir=$out/bin --docdir=$doc/share/doc/$pname --with-gdalconfig=${gdal}/bin/gdal-config --with-jsondir=${json_c.dev}"
```

Comment:
I think `$pname` here needs to be `${pname}` since it's coming from nix and not a shell var. (same for below)